### PR TITLE
feat(ccd): added new component

### PR DESCRIPTION
**Pull request by agliga**
## Description
* Added CCD component

## Context
* We need to toggle between one and two figures in this. Added some boolean types to allow toggling

## References
https://github.com/eBay/ebayui-core/issues/2326

### CODE REVIEW

**Comment by agliga** [comment]
Test is failing because we havent linked up skin. Should pass once we have the correct skin version.

**Review by ArtBlue — APPROVED** [review]
LGTM!

File: src/components/ebay-ccd/index.marko [hunk 1]
```diff
@@ -0,0 +1,61 @@
+import { processHtmlAttributes } from "../../common/html-attributes";
+import type { WithNormalizedProps } from "../../global";
+
+static interface CCDInput extends Omit<Marko.Input<"div">, `on${string}`> {
+    max?: string;
+    min?: string;
+    "charger-icon"?: string;
```
Comment (LuLaValva) [inline src/components/ebay-ccd/index.marko:7]: This should be an enum, like `"none" | "included" | "notIncluded"`

Comment (LuLaValva) [reply src/components/ebay-ccd/index.marko:7]: Also, I think we generally use kebab-case for attributes like this. Is `"notIncluded"` standard terminology or was it chosen arbitrarily?

File: src/components/ebay-ccd/index.marko [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    secondaryText = "USB PD",
+    ...htmlInput
+} = input;
+$ (input as any).toJSON = noop;
+
+$ const hasFastCharging = secondaryType === "usbpd";
+
+$ const noIcon = chargerIcon !== "included" && chargerIcon !== "notIncluded";
+$ const hasCharger = chargerIcon === "included";
+$ const chargerA11yText = noIcon ? "" : `Charger${hasCharger ? " " : "not "} included.`;
```
Comment (LuLaValva) [inline src/components/ebay-ccd/index.marko:35]: This is missing a space
```suggestion
$ const chargerA11yText = noIcon ? "" : `Charger${hasCharger ? " " : " not "} included.`;
```

**Review by LuLaValva — COMMENTED** [review]
Just a few minor comments

**Pull request merged**